OpenRA.Game/Map/Map.cs

```diff
@@ -493,7 +493,8 @@ public void Save(string toPath)
 			}
 
 			// Update existing package
-			Container.Write(entries);
+			if (Container != null)
```

Comment:
A better fix would probably to add an `|| Container == null` case to the conditional above.